carmin-server/server/resources/executions.py

```diff
@@ -18,13 +19,32 @@ class Executions(Resource):
     @get_db_session
     @marshal_response(ExecutionSchema(many=True))
     def get(self, user, db_session):
+        offset = request.args.get('offset')
```

Comment:
Please use the type parameter. https://stackoverflow.com/questions/34671217/in-flask-what-is-request-args-and-how-is-it-used